entries/jQuery.parseHTML.xml

```diff
@@ -17,6 +17,7 @@
   <longdesc>
     <p><code>jQuery.parseHTML</code> uses a native DOM element creation function to convert the string to a set of DOM elements, which can then be inserted into the document.</p>
     <p>By default, the <code>context</code> is the current <code>document</code> if not specified or given as <code>null</code> or <code>undefined</code>. If the HTML was to be used in another document such as an iframe, that frame's document could be used.</p>
+    <p>Worth noting is that <code>jQuery.parseHTML()</code>, unlike the standard jQuery selection methods, will actually render any leading or trailing text in the HTML string as text nodes (even if that's only whitespace). To prevent trailing/leading whitespace from being converted to text nodes you can pass the HTML string through <a href="/jQuery.trim/"><code>jQuery.trim</code></a>.</p>
```

Comment:
I'd drop the comparison to selection (which this is independent of) and move the meat of this paragraph to the end of the first (e.g., "to prevent trailing/leading whitespace from being appearing in text nodes, pass the HTML string through jQuery.trim").